src/index.html

```diff
@@ -80,8 +80,13 @@ <h1 id="clock" class="time mb+ text-center mt">-:-</h1>
         </a>
       </div>
 
+      <div class="toggle-switch text-center mb+">
+        <label> <input type="radio" name="crypto" value="bitcoin" checked /> Bitcoin </label>
+        <label> <input type="radio" name="crypto" value="ethereum" /> Ethereum </label>
+      </div>
```

Comment:
I see your point, and I tend to like it at one side, but on another - I prefer much more to see this in the future https://github.com/superKalo/crypto-tab/issues/54 , instead as a playable call-to-action here.